backend/nlp/handler.py

```diff
@@ -3,17 +3,26 @@
 import json
 import imp
 import sys
+import os
 
 sys.modules["sqlite"] = imp.new_module("sqlite")
 sys.modules["sqlite3.dbapi2"] = imp.new_module("sqlite.dbapi2")
 
 #pylint: disable=unused-import
```

Comment:
Is there any way we can eliminate some of these pylint statements? If they're all a result of our weird NLTK imports we can leave them, but I'd like to keep to as few as possible.

Reply:
I believe it's due to all of the groupings and what it expects for imports, I can double check but I think they're neccesary